### DISCUSSION TITLE
Tank Scaling

### DISCUSSION THREAD

**Discussion by ZaamFast** [scuffed56/SCUFFED5.0#68] [Ideas]
I think Scuffed and Road to 56 are to infantry based with the technologies given in game. To counter act this there are a number of ways to nerf infantry or buff tanks. Personally I think infantry should be nerfed by removing flamethrower and sniper technologies as it heavily inflates the stats of infantry. Second I think tanks should get better with each model researched as currently the only real improvements are reliability and armor. To increase the stats of breakthrough, speed, or defense could see an increased use in newer tank models. Tank cannons also seem to be missing as the medium tank has no cannon 3. This makes it so while the infantry technologies scales a large margin with each artillery bonus tanks stay stagnant. 

TLDR: 
Add scalable tanks
Nerf the excessive infantry bonuses.
